Apps/Src/Task_Amperes.c

```diff
@@ -45,7 +46,7 @@ void Task_AmperesMonitor(void *p_arg) {
 
 		//Send measurement to CAN queue
 		int current = Amps_GetReading();
-		CanData.f = (float)current/1000; //send data in Amps
+		memcpy(&(CanData.w), &current, sizeof(CanData.w));
```

Comment:
Is there any reason in particular to be doing a memcpy right here, instead of an assignment?

Reply:
`CanData.w` is unsigned, but `current` is signed, so I don’t think we can typecast safely if we have a negative `current`

Reply:
I looked into this some more, and I think we can safely do this with a cast, so I'll change it